### ISSUE TITLE
Disable container?

### ISSUE THREAD

**Issue by focussing** [karolis-sh/tailwind-bootstrap-grid#125]
This is a great lib!!

The docs state the following:
_* NOTE: When using the .container class from this plugin you will need to [disable](https://tailwindcss.com/docs/container#disabling-entirely) the core [container plugin](https://tailwindcss.com/docs/container/) as both plugins expose a .container class._

I tried to find that for Tailwind 3 and saw [this] (https://redpixelthemes.com/blog/tailwindcss-container-component). It basically says that the container class will not be generated if it's not used. But when I use it via the tailwind-bootstrap-grid would the tailwind container be generated?
How shoudl we deal with this?

**Comment by karolis-sh** [comment]
It used to be like so in tailwind config:
```
module.exports = {
  corePlugins: {
    container: false,
  },
};
```

**Comment by focussing** [comment]
Yes I know but then I read this (https://redpixelthemes.com/blog/tailwindcss-container-component/#disable-the-container-in-older-tailwind-css-versions) and that's why I asked my question

**Comment by focussing** [comment]
I tested the tailwind config addition and it works, also the warning about double container classes is gone.
Thank you!
